src/flows/components/header/index.tsx

```diff
@@ -189,21 +188,14 @@ const FlowHeader: FC = () => {
     }
   }
 
-  const hideShare = () => {
-    setSharing(false)
-  }
-
-  const deleteShare = () => {
-    setLinkDeleting(RemoteDataState.Loading)
```

Comment:
how come no user more user feedback?

Reply:
just moved the user feedback to overlay component